src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -234,7 +268,8 @@ public async Task InitializeAsync(IEnumerable<FunctionMetadata> functions, Cance
 
             _workerRuntime = _workerRuntime ?? Utility.GetWorkerRuntime(functions, _environment);
 
-            if (string.IsNullOrEmpty(_workerRuntime) || _workerRuntime.Equals(RpcWorkerConstants.DotNetLanguageWorkerName, StringComparison.InvariantCultureIgnoreCase))
+            // In case of multi language runtime, _workerRuntime has no significance, thus skipping this check for multi language runtime environment
+            if ((string.IsNullOrEmpty(_workerRuntime) || _workerRuntime.Equals(RpcWorkerConstants.DotNetLanguageWorkerName, StringComparison.InvariantCultureIgnoreCase)) && !_environment.IsMultiLanguageRuntimeEnvironment())
```

Comment:
Curious, how are we handling the case where one of the languages in multi-lang is Dotnet? As I am seeing different handling for Dotnet here.

Reply:
Dotnet is running as in-proc function and does not create and workers. I have added test to call C#, Java, Node function. Let me know if i am missing something!

Reply:
I suppressed it because in case of Multi Language, `_workerRuntime` is alwys `""`